doc/modules/impute.rst

```diff
@@ -50,5 +50,35 @@ Note that, here, missing values are encoded by 0 and are thus implicitly stored
 in the matrix. This format is thus suitable when there are many more missing
 values than observed values.
 
-:class:`SimpleImputer` can be used in a Pipeline as a way to build a composite
-estimator that supports imputation. See :ref:`sphx_glr_auto_examples_plot_missing_values.py`.
\ No newline at end of file
+A more sophisticated approach is to use the :class:`MICEImputer` class, which
```

Comment:
Let's give this a heading: Multivariate Imputation. Perhaps also add a heading for Univariate Imputation above.